bundle/manifests/operator.ibm.com_authentications.yaml

```diff
@@ -228,6 +228,10 @@ spec:
                     type: string
                   providerIssuerURL:
                     type: string
+                  auditURL:
```

Comment:
These changes should also be happening in the config directory, and changes here should be generated with `make bundle`